generate.sh

```diff
@@ -23,35 +23,6 @@ find src -type f -exec sed -i '/The version of the OpenAPI document/d' {} \;
 
 # Cookie storage
 sed -i 's/Client::new()/Client::builder().cookie_store(true).build().unwrap()/g' src/apis/configuration.rs
-
-# https://github.com/OpenAPITools/openapi-generator/issues/14171
-# Replace Option<SortOption with Option<crate::models::SortOption in src/apis
-sed -i 's/Option<SortOption>/Option<crate::models::SortOption>/g' src/apis/*.rs
-# Replace Option<ReleaseStatus with Option<crate::models::ReleaseStatus in src/apis
-sed -i 's/Option<ReleaseStatus>/Option<crate::models::ReleaseStatus>/g' src/apis/*.rs
-# Replace Option<OrderOption with Option<crate::models::OrderOption in src/apis
-sed -i 's/Option<OrderOption>/Option<crate::models::OrderOption>/g' src/apis/*.rs
-# Replace message_type: InviteMessageType with message_type: crate::models::InviteMessageType in src/apis
-sed -i 's/message_type: InviteMessageType/message_type: crate::models::InviteMessageType/g' src/apis/*.rs
-# Replace Option<GroupSearchSort with Option<crate::models::GroupSearchSort in src/apis
-sed -i 's/Option<GroupSearchSort>/Option<crate::models::GroupSearchSort>/g' src/apis/*.rs
-
-# -Missing Github Issue-
-# Append patches/InviteMessageType-Display.rs to invite_message_type.rs
-cat patches/InviteMessageType-Display.rs >> src/models/invite_message_type.rs
-# Remove the ToString section and it's included function
-# impl ToString for InviteMessageType {
-#     fn to_string(&self) -> String {
-#         match self {
-#             Self::Message => String::from("message"),
-#             Self::Response => String::from("response"),
-#             Self::Request => String::from("request"),
-#             Self::RequestResponse => String::from("requestResponse"),
-#         }
-#     }
-# }
-
-
-sed -z -i 's/impl ToString for InviteMessageType {\n[ a-zA-Z_\(\)&-\>{\n:=",]*}\n    }\n}//g' src/models/invite_message_type.rs
```

Comment:
Can you confirm these patches are no longer necessary?

Reply:
Sure:
- There are no compilation errors. 
- The display is now generated: https://github.com/vrchatapi/vrchatapi-rust/blob/496172375f13d8c2d12a36739aa72e389816b7e4/src/models/invite_message_type.rs#L26-L36

Though i dont have a piece of code I can test this on, someone else would need to do an integration test.